### DISCUSSION TITLE
No push notification for new releases at watched repo at iOS

### DISCUSSION THREAD

**Discussion by st1ng7ay** [community/community#38669] [Mobile]
I have some watched repo ( or starred, or add to list ).
How to receive push notifications to iOS ?

emails are coming, notifications come to inbox, but no push notifications at iPhone
no such notification types:

![example](https://user-images.githubusercontent.com/79450231/200841264-674a25c6-00fb-4845-922a-58a7354b9ec0.jpg)

**Comment by aconsuegra** [comment]
Thanks for your feedback @st1ng7ay! We currently don't support push notifications for releases, but I can clearly see the benefit of supporting it, so I am going to create an issue in our backlog to track this request.

**Reply by st1ng7ay** [reply]
Thank you for that.
No push for discussions also.

**Reply by Negniq-6cehvi-futwiv** [reply]
Roles in an organization
